website/_site/search.json

```diff
@@ -109,6 +109,68 @@
     "href": "about.html#steering-committee",
     "title": "About",
     "section": "",
+<<<<<<< Updated upstream
```

Comment:
I still see this marks (now in the json file). Not sure what is going on but it seems related to "Stashed changes" (from the closing >>>>>)

Reply:
I don't see this on the html rendering 🤷‍♀️

Reply:
I just had a similar issue on PR #9 that I fixed in the GH editor. I think `git stash` is being weird on my end too! we can work through this during our Thursday meeting if you haven't resolved this yet 💪

Reply:
Then all good :D